src/dataregistry_cli/cli.py

```diff
@@ -3,9 +3,34 @@
 import argparse
 from dataregistry.db_basic import SCHEMA_VERSION
 from .register import register_dataset
+from .delete import delete_dataset
 from .query import dregs_ls
 from dataregistry.schema import load_schema
 
+def _add_generic_arguments(parser_obj):
+    """
+    Most commands have the schema/root_dir/etc as options, have a function that
```

Comment:
Most commands have the schema, root_dir, etc. as options. This function does that for us.